.drone.star

```diff
@@ -216,6 +216,7 @@ config = {
             "extraEnvironment": {
                 "REMOTE_BACKEND_HOST": "http://federated",
                 "EXPECTED_FAILURES_FILE": "%s/tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md" % dir["web"],
+                "WEB_UI_CONFIG": "%s/dist/config.json" % dir["web"],
```

Comment:
Instead of repeating, you could add a default env like this that would get overridden if provided in extraEnvironment
```
environment["WEB_UI_CONFIG"] = "%s/dist/config.json" % dir["web"]
```
above this line
https://github.com/owncloud/web/blob/12a449478cc66709a2f5150788104ca2c63f326c/.drone.star#L2413-L2414

Reply:
@saw-jan bro then how about removing all `"WEB_UI_CONFIG": "%s/dist/config.json" % dir["web"],` from extraEnvironment  in acceptance that way it provide consistency.